src/Resolver/AggregateResolver.php

```diff
@@ -53,13 +53,24 @@ public function attach(ResolverInterface $resolver, int $priority = 1): self
     public function resolve(string $name): string
     {
         foreach ($this->queue as $resolver) {
-            try {
-                return $resolver->resolve($name);
-            } catch (TemplateCannotBeFound) {
+            if (! $resolver->has($name)) {
```

Comment:
This is now worse, because we do double the I/O 🤣

Sorry!

Reply:
I/O is only happening in the PathStack variant.

What do you think is a good solution here?

Internal in-memory cache? Or changing `Resolver::get(string): string|false`?

Reply:
> I/O is only happening in the PathStack variant.

Which is the most common one, heh

I think sticking to `|false` is the least evil change right now